.travis.yml

```diff
@@ -25,10 +19,11 @@ before_script:
   - start-stop-daemon --start --background --exec $(pwd)/lighthouse-core/scripts/launch-chrome.sh
   - sleep 5
 script:
+  - if [[ $(node -v) =~ ^v4.* ]]; then npm config set harmony=--harmony; else npm config rm harmony; fi
```

Comment:
if instead of setting an npm variable we just `export __node_harmony=true` then no npmrc or `sed /undefined//` required. wdyt?